guides/handGrenades.md

```diff
@@ -0,0 +1,107 @@
+# Hand Grenade Guide
+
+## Role Description
+
+All operators should be expected to familiarize themselves with the use of hand grenades. This guide will cover mainly fragmentation grenades, but will briefly cover the other 2 types as well.
+
+## Specialized Equipment
+
+Completing the certification for hand grenades will unlock the usage of fragmentation grenades as well as flashbang grenades.
+
+## Usage
+
+### Callouts
+
+Similar to throwing a grenade, every time you toss a grenade, a call should be made over proximity voice. The call should be in the format:
+
+> [Frag|Flash|Smoke] out. [Direction]
+
+Examples:
+
+> Frag Out! North!
+> Flash Out! South-East!  
+> Smoke Out! West! 
+
+The call out should not slow down your ability to respond to threats. At a minimum you must always call out "[Grenade Type] Out!"
+
+If the employment of the grenade fails, the thrower should immediately alert the other operators in his vicinity.
```

Comment:
```suggestion
If the employment of the grenade fails, the thrower should immediately alert the other operators in his vicinity. When calling a bad grenade toss, the direction **must** be included in the call.
```

Reply:
I mean we can include the direction but maybe just include it n the throw call then? By the time you say "Southwest" the nade will have exploded.